assets/dev/scss/admin/admin-top-bar.scss

```diff
@@ -0,0 +1,173 @@
+@import "../../../../assets/dev/scss/helpers/direction";
+@import "../../../../assets/dev/scss/helpers/variables";
+@import "../../../../assets/dev/scss/helpers/mixins";
+@import "../../../../assets/dev/scss/editor/breakpoints";
+
+$top-bar-height: 50px;
```

Comment:
Why is the code here?
you should include the file at `modules/admin-top-bar/assets/scss/admin.scss`

Reply:
Done.